recipes/octo-keygen-cpp/all/test_v1_package/CMakeLists.txt

```
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 3.8)
+project(test_package LANGUAGES CXX)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup(TARGETS)
+
```

```suggestion

find_package(octo-keygen-cpp CONFIG REQUIRED)

```
generated by cmake_find_package_multi

---

fixed :)